src/note/mod.rs

```diff
@@ -52,6 +51,8 @@ pub struct Note {
     pub tags: Vec<String>,
     /// The date assigned to the note.
     pub date: Option<OffsetDateTime>,
+    /// Other metadata from the front matter, kept as-were
+    pub metadata: Metadata,
```

Comment:
Perhaps we should consider another name for this field? `metadata` might be a bit confusing, since the other fields are metadata as well?

My suggestion is something like `other`, or `rest`? What do you think?

Reply:
I see your point, [zola] calls them `extra`. Could this work for you?

[zola]: https://www.getzola.org/documentation/templates/pages-sections/#page-variables

Reply:
Sure, that sounds reasonable!